conf/app.routes

```diff
@@ -106,6 +106,7 @@ GET         /my-organisation/appoint-new-agent/multiple-properties
 POST        /my-organisation/appoint-new-agent/multiple-properties                                                        @controllers.agentAppointment.AgentRelationshipController.submitMultipleProperties()
 
 GET         /my-organisation/appoint-new-agent/checkYourAnswers                                                           @controllers.agentAppointment.AgentRelationshipController.checkAnswers()
+POST        /my-organisation/appoint-new-agent/appointmentChanges                                                         @controllers.agentAppointment.AgentRelationshipController.submitAppointmentChanges()
```

Comment:
as this method is used for revoke as well, we cannot call this appoint.

Reply:
good point